cli/cmd/cluster.go

```diff
@@ -188,6 +188,8 @@ func (c controlplaneUpdater) upgradeComponent(component string) {
 
 	update := action.NewUpgrade(actionConfig)
 
+	update.Atomic = true
+
```

Comment:
Why not squash this in previous commit that added the code?

Reply:
Fixed.